### PR TITLE
Implement spiral-matrix exercise

### PR DESCRIPTION
**Pull request by jpreese**

### CODE REVIEW

**Comment by jpreese** [comment]
Give me a sec, I'm honestly not sure why the build can't find the GetMatrix method at the moment.

**Comment by jpreese** [comment]
Ok, I dug through the Cake script and saw it was failing after replacing the implementation with the exercise stub. I had my class named Example to match Example.cs, but I see its renaming that file behind the scenes and expects the Example class to be named the same as the exercise. So that's been updated.

File: exercises/spiral-matrix/SpiralMatrixTest.cs [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        public void Spiral_of_size_5()
+        {
+            var spiral = new int[,] 
+            { 
+                { 1, 2, 3, 4, 5 }, 
+                { 16, 17, 18, 19, 6 }, 
+                { 15, 24, 25, 20, 7 }, 
+                { 14, 23, 22, 21, 8 }, 
+                { 13, 12, 11, 10, 9 }
+
```
Comment (ErikSchierboom) [inline exercises/spiral-matrix/SpiralMatrixTest.cs]: Could you remove this empty line?

**Comment by ErikSchierboom** [comment]
Brilliant work once again @jpreese! Thanks a lot :tada:

**Pull request merged**